app/assets/javascripts/bootstrap-json/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bootstrap-json",
```

Comment:
I would like to rename this package to `discourse-ember-cli-proxy`, but will do it in a follow-up PR so that the diff is easier to read